src/features/stake/helpers/utils.ts

```diff
@@ -2,10 +2,12 @@ import { id } from 'ethers'
 import type { BaseTransaction } from '@safe-global/safe-apps-sdk'
 
 const WITHDRAW_SIGHASH = id('requestValidatorsExit(bytes)').slice(0, 10)
+const CLAIM_SIGHASH = id('batchWithdrawCLFee(bytes)').slice(0, 10)
 
 export const getStakeTitle = (txs: BaseTransaction[] | undefined) => {
   const hashToLabel = {
     [WITHDRAW_SIGHASH]: 'Withdraw request',
+    [CLAIM_SIGHASH]: 'Claim',
```

Comment:
Can’t these titles be deduced from the txInfo/txConfirmation type?

Reply:
As discussed in the call - we currently do this because the app title is a component that is rendered before the modal content and as such it doesn't have access to the txInfo. This would need a refactoring (since you guys are working on the txflow #4157 you can consider it there)